utility/ATTiny/RF24_arch_config.h

```diff
@@ -6,41 +7,40 @@
 #ifndef __RF24_ARCH_CONFIG_H__
 #define __RF24_ARCH_CONFIG_H__
 
-/*** USER DEFINES:  ***/
-//#define FAILURE_HANDLING
-//#define MINIMAL
-/**********************/
-
-#define rf24_max(a, b) (a>b?a:b)
-#define rf24_min(a, b) (a<b?a:b)
-
-#if ARDUINO < 100
-
-    #include <WProgram.h>
-
-#else
-    #include <Arduino.h>
-#endif
-
-#include <stddef.h>
-
-// Include the header file for SPI functions ( Main SPI code is contained in RF24.cpp for simplicity )
-#include "spi.h"
-
-#define _SPI SPI
-
-#ifdef SERIAL_DEBUG
-    #define IF_SERIAL_DEBUG(x) ({x;})
-#else
-    #define IF_SERIAL_DEBUG(x)
-    #if defined(RF24_TINY)
-        #define printf_P(...)
+  /*** USER DEFINES:  ***/  
+  //#define FAILURE_HANDLING
+  //#define MINIMAL
+  /**********************/
+
+  #define rf24_max(a,b) (a>b?a:b)
+  #define rf24_min(a,b) (a<b?a:b)
```

Comment:
Please put the space back in the `(a, b)` snippets. Quite a bit of work went into formatting these source files according to [NASA C-style guidelines](https://ntrs.nasa.gov/citations/19950022400) and no space after commas is a direct violation of section 2.2.2 about spacing.

Reply:
Weird. I never (knowingly) changed that file's formatting. In any case, I've updated that file now.